.github/workflows/close-inactive-issues.yml

```diff
@@ -5,6 +5,27 @@ on:
   workflow_dispatch: # allow manual trigger
 
 jobs:
+  close-issues-in-triage:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          operations-per-run: 1000
+          days-before-issue-stale: 28
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 28 days with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for 14 days since being marked as stale."
+          close-issue-reason: 'not_planned'
+          days-before-pr-stale: -1 # ignore PRs (overwrite default days-before-stale)
+          days-before-pr-close: -1 # ignore PRs (overwrite default days-before-close)
+          remove-issue-stale-when-updated: true
+          exempt-all-issue-milestones: false # issues with assigned milestones will be ignored
+          only-label: 'triage'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
   close-issues-with-assignee:
```

Comment:
should the existent "close issues" rules being removed? Only the `triage` issues should be automatically closed. (we'd need to put the `triage` label manually on already open ones)

Reply:
I thought about it, and I came out thinking that maybe auto-stale-ing issues that are not assigned to a milestone could be a safe guard, because these issues probably are not very relevant to anyone, especially if they don't have an assignee.

How about we increase the time from 28 days to, say, 90 days and remove the exclusion for `bug` issues? we could also merge the `-with-assignee` and `-without-assignee` for all I care.

Reply:
the major difference would be in _who created them_, and the time after which they get stale'd and ultimately closed.